icicle-compiler/src/Icicle/Repl.hs

```diff
@@ -93,6 +93,8 @@ getInitialState = do
                 []
               UseColor ->
                 [FlagColor]
+
+      , stateCFlags = "-O0 -UICICLE_ASSERT"
```

Comment:
Could this be `-DICICLE_ASSERT=0` instead?

Reply:
nah, gcc 4.7 barfs if you pass the same identifier multiple times as arguments like "-DA=1 -DA=2"

Reply:
wow, that's delightful

Reply:
real helpful